### ISSUE TITLE
Files with the word "await" in them disappear from coverage

### ISSUE THREAD

**Issue by danias** [istanbuljs/nyc#884]
## Expected Behavior
Files should appear in the coverage report
## Observed Behavior
Files are not there if they contain the word await (even if the file only contains that word!)
## Forensic Information

**Operating System:** macOS and Linux
**Environment Information:** information about your project's environment, see instructions below:
Have tried with both node 6.14 and node 9

[output.txt](https://github.com/istanbuljs/nyc/files/2166247/output.txt)

**Comment by coreyfarrell** [comment]
I use NYC with plenty of repositories that use the async/await keywords.  Please link a repository demonstrating this issue.

**Comment by danias** [comment]
I figured out what was going on. The problem was the await is a reserved word and in ES5 code where I was requiring `var await = require('asyncawait/await');` where await is [is how it is usually named](https://www.npmjs.com/package/asyncawait#asyncawait-101) and then using it in my code. To bypass this, I had to say `var Await = require('asyncawait/await');`. Of course, await [is not a reserved word in ES5](https://mathiasbynens.be/notes/reserved-keywords) as it was introduced in ES6.
